src/updates_explorer/bot_listener.ts

```diff
@@ -66,20 +61,17 @@ export class BotListener {
           "chat_member",
           "chat_join_request",
         ],
-        onStart: async () => {
-          this.state = "listening";
-        },
+        onStart: async () => {},
```

Comment:
Can you do this to all empty blocks?

```suggestion
        onStart: async () => {
          //
        },
```

Reply:
Sure thing